modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1521,6 +1521,24 @@ public BlogsEntry updateStatus(
 				_pingTrackbacks(
 					entry, trackbacks, pingOldTrackbacks, serviceContext);
 			}
+
+			// Resources
+
+			if (GetterUtil.getBoolean(
+					serviceContext.getAttribute("showPermissions"))) {
```

Comment:
Where is this service context attribute initialized? Are you sure this is correct?

Reply:
It works. When I tried to make the check null safe the source formating reseted back to this form.